projects/kit/src/lib/masks/time/utils/tests/parse-time.spec.ts

```diff
@@ -0,0 +1,141 @@
+import {describe, expect, it} from '@jest/globals';
+import type {MaskitoTimeMode} from '@maskito/kit';
+import {maskitoParseTime} from '@maskito/kit';
+
+describe('maskitoParseTime', () => {
+    const testCases = new Map<MaskitoTimeMode, Array<{ms: number; text: string}>>([
+        [
+            'HH:MM:SS.MSS',
+            [
+                {ms: 0, text: ''},
+                {ms: 0, text: '00:00:00.000'},
+
+                {ms: 36000000, text: '1'},
```

Comment:
**Nitpicking:**

Could you change order of objects' properties?

```suggestion
                {text: '', ms: 0},
                {text: '00:00:00.000', ms: 0},

                {text: '1', ms: 36000000},
```

It is more easier to read when order is "from => to"

Reply:
Done.